image/templates/helm/stackrox-secured-cluster/values-scanner-v4-indexer.yaml.example

```diff
@@ -0,0 +1,104 @@
+# # NOTE:
+# # The Scanner v4 Indexer is not available in the secured clusters installed via roxctl.
```

Comment:
Can we note that Scanner v4 Matcher is never available?

Reply:
done